recipes/itk/all/conanfile.py

```diff
@@ -0,0 +1,217 @@
+import os
+import re
+import shutil
+
+from conans import ConanFile, CMake, tools
+from fnmatch import fnmatch
+
+
+class LibITKConan(ConanFile):
+    name = "itk"
+    upstream_version = "4.13"
+    upstream_patch = "2"
+    version = "{0}.{1}".format(upstream_version, upstream_patch)
+
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False]}
+
+    default_options = {"shared": True}
+
+    homepage = "http://www.itk.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "http://www.itk.org/licensing/"
+    description = "Insight Segmentation and Registration Toolkit"
+    _source_subfolder = "sf"
+    build_subfolder = "build"
+    short_paths = True
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if 'CI' not in os.environ:
+            os.environ["CONAN_SYSREQUIRES_MODE"] = "verify"
+
+    def requirements(self):
+
+        self.requires("libjpeg-turbo/2.0.2")
+        self.requires("expat/2.2.7")
+        self.requires("libtiff/4.0.9")
+        self.requires("libpng/1.6.37")
+        self.requires("libjpeg/9c")
+        self.requires("zlib/1.2.11")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ITK-{0}.{1}".format(
+            self.upstream_version,
+            self.upstream_patch),
+            self._source_subfolder)
+
+    def build(self):
+        itk_source_dir = os.path.join(
+            self.source_folder,
+            self._source_subfolder)
+
+        cmake = CMake(self)
+
+        cmake.definitions["BUILD_EXAMPLES"] = "OFF"
+        cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
That one should be automatically set by the CMake build helper, you don't need to set it by hand.

Reply:
done